packages/babel-parser/src/parser/expression.js

```diff
@@ -758,8 +758,13 @@ export default class ExpressionParser extends LValParser {
     optional: boolean,
   ): N.Expression {
     if (node.callee.type === "Import") {
-      if (node.arguments.length !== 1) {
-        this.raise(node.start, Errors.ImportCallArity);
+      const maxArity = this.hasPlugin("moduleAttributes") ? 2 : 1;
+      if (node.arguments.length === 0 || node.arguments.length > maxArity) {
+        this.raise(
+          node.start,
+          Errors.ImportCallArity,
+          this.hasPlugin("moduleAttributes") ? "two arguments" : "one argument",
```

Comment:
we are passing a message here to explain the exact number of arguments needed. for module attributes it is two arguments and for every other thing it is only one argument.